tests/ui/missing_intopy.stderr

```diff
@@ -0,0 +1,38 @@
+error[E0599]: the method `into_py_result` exists for struct `Blah`, but its trait bounds were not satisfied
+   --> tests/ui/missing_intopy.rs:3:1
+    |
+1   | struct Blah;
+    | ------------
+    | |
+    | method `into_py_result` not found for this
+    | doesn't satisfy `Blah: IntoPy<Py<PyAny>>`
+    | doesn't satisfy `Blah: IntoPyResult<Blah>`
+2   |
+3   | #[pyo3::pyfunction]
+    | ^^^^^^^^^^^^^^^^^^^ method cannot be called on `Blah` due to unsatisfied trait bounds
+    |
+    = note: the following trait bounds were not satisfied:
+            `Blah: IntoPy<Py<PyAny>>`
+            which is required by `Blah: IntoPyResult<Blah>`
+note: the following trait must be implemented
+   --> src/conversion.rs
+    |
+    | / pub trait IntoPy<T>: Sized {
+    | |     /// Performs the conversion.
+    | |     fn into_py(self, py: Python<'_>) -> T;
+    | | }
+    | |_^
+    = note: this error originates in the attribute macro `pyo3::pyfunction` (in Nightly builds, run with -Z macro-backtrace for more info)
+
+error[E0277]: the trait bound `Blah: IntoPyCallbackOutput<_>` is not satisfied
+   --> tests/ui/missing_intopy.rs:3:1
+    |
+3   | #[pyo3::pyfunction]
+    | ^^^^^^^^^^^^^^^^^^^ the trait `IntoPyCallbackOutput<_>` is not implemented for `Blah`
+    |
+note: required by a bound in `pyo3::callback::convert`
+   --> src/callback.rs
+    |
+    |     T: IntoPyCallbackOutput<U>,
+    |        ^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `pyo3::callback::convert`
+    = note: this error originates in the attribute macro `pyo3::pyfunction` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
It's unfortunate this `IntoPyCallbackOutput` part is back - but overall I think it's worth it. The first part of the error is very very good.